scripts/network-init

```diff
@@ -28,13 +28,31 @@ prepare_networking() {
 
     if [ -e ${MANAGEMENT_CONF} ] ; then
         source ${MANAGEMENT_CONF}
-        if PHYSICAL_PIF="$($XE pif-list host-uuid=${INSTALLATION_UUID} device=${LABEL} params=uuid --minimal)" ; then
+        if [ ! -z "${BOND_MODE}" ] ; then
+            BOND_NETWORK="$($XE network-create name-label='Bonded pool-wide network with mode '${BOND_MODE})"
+            readarray -td, a <<<${BOND_MEMBERS}, ; unset 'a[-1]' ;
+            for MEMBER_DEVICE in "${a[@]}"
```

Comment:
Is this correct? `"${a[@]}"` is a single word and hence there is no reason to use `for`

Reply:
Are you sure, I have tested it and it iterates on each device in `BOND_MEMBERS`

Reply:
I'm not sure but `for x in in "a b c"; do echo $x; done` would emit `a b c` in one line.

Reply:
```bash
BOND_MEMBERS="eth0,eth1,xenbr0"

readarray -td, a <<<${BOND_MEMBERS}, ; unset 'a[-1]' ;
for MEMBER_DEVICE in "${a[@]}"
do :
    echo ${MEMBER_DEVICE}
done
```
resuts in
```
eth0
eth1
xenbr0
```

The readarray creates an array from the sring `BOND_MEMBERS` and then its iteraded by the for loop